package.json

```diff
@@ -18,9 +18,15 @@
     "url": "https://github.com/nervnews/nerv/issues"
   },
   "homepage": "https://github.com/nervnews/nerv#readme",
+  "devDependencies": {
+    "morgan": "^1.9.0",
+    "nodemon": "^1.14.11"
+  },
   "dependencies": {
     "body-parser": "^1.18.2",
+    "emotional": "0.0.3",
```

Comment:
what is this !!?

Reply:
module for sentiment analysis!

Reply:
https://github.com/nervnews/nerv/wiki/Sentiment-Analysis